website/source/docs/agent/encryption.html.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: "docs"
+page_title: "Encryption"
+sidebar_current: "docs-agent-encryption"
+description: |-
+  Learn about the encryption in Nomad.
+---
+
+The Nomad agent supports encrypting all of its network traffic. There are
+two separate encryption systems, one for gossip traffic and one for RPC.
+
+# Gossip Encryption
+
+Enabling gossip encryption only requires that you set an encryption key when
+starting the Nomad server agent. The key can be set via the [`encrypt`](/docs/agent/config.html#encrypt) parameter: the
```

Comment:
I think we can drop "agent" from "Nomad server agent". "When starting the Nomad server" is clear.